src/storage-adapters/git/git.js

```diff
@@ -1,21 +1,26 @@
+import fsApi from 'fs';
 import path from 'path';
 
-import config from 'config';
 import simpleGit from 'simple-git';
 
 process.env.LC_ALL = 'en_GB'; // Ensure git messages will be in English as some errors are handled by analysing the message content
 
+const fs = fsApi.promises;
+
 export default class Git {
-  constructor(repositoryPath) {
+  constructor({ path: repositoryPath, author }) {
     this.path = repositoryPath;
-    this.git = simpleGit(repositoryPath, { maxConcurrentProcesses: 1 });
+    this.author = author;
   }
 
-  async init() {
-    return this.git.init();
-  }
+  async initialize() {
+    if (!await fileExists(this.path)) {
```

Comment:
As it's not a file but a directory, maybe using `existsSync` would be better

Reply:
I though it was deprecated but `exists` is, not `existsSync`. So :+1: